app/template.html

```diff
@@ -45,7 +45,7 @@ <h1>
                       <li class="dropdown">
                         <a href="" class="dropdown-toggle" data-toggle="dropdown"><i class="glyphicon glyphicon-user"></i><span ng-bind="sessionUser"></span><b class="caret"></b></a>
                         <ul class="dropdown-menu animated fadeInUp">
-                          <li><a href="#!/profile">Profile</a></li>
+                          <li ng-if="sessionProvider !== 'keycloak'"><a href="#!/profile">Profile</a></li>
```

Comment:
Hmm, there are some setting in a users profile that we still need to be able to access. Like setting their reporting preferences and filtering preferences. We will still need to view this profile page, however, we won't be able to change certain fields.

Reply:
cool done :)